src/components/navigation/navigation/navigation.stories.ts

```diff
@@ -112,7 +112,7 @@ const navigationActionsL = (): TemplateResult => html`
   <sbb-navigation-button id="nav-1" data-testid="navigation-section-trigger-1">
     Tickets & Offers
   </sbb-navigation-button>
-  <sbb-navigation-button id="nav-2" class="sbb-active"
+  <sbb-navigation-button id="nav-2" class="sbb-active" aria-current="page"
```

Comment:
should the navigation section stories be updated as well? and is there not one home story which has an sbb-active without aria-current? (not sure)